qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -92,7 +90,10 @@ def __init__(
 
         # Set fixed options
         self._full_sampling = full_sampling
-        self._clifford_utils = CliffordUtils()
+        self._transpiled_cliff_circuits = {}
+        # transpiled clifford circuits for 1 and 2 qubits respectively
+        self._transpiled_cliff_circuits[1] = None
+        self._transpiled_cliff_circuits[2] = None
```

Comment:
This is also problematic because in principle we should be able to run 3Q+ RB though its outcome might not be statistically confident.

Reply:
I could of course define this as an array. But I preferred to write it this way to stress that for now only 1 and 2 qubit -rb are supported. Many changes will be needed to support more than 2 qubits. One option is to keep the legacy code for 3 or more qubits, but I am not sure if that code worked for more than 2 qubits. @ShellyGarion - do you know if the previous version worked in this case?

Reply:
Then this should be implemented as a subclass of `StandardRB`, i.e. `StandardRB1Q` and `StandardRB2Q` (because constructor argument is `qubits: Sequence[int]`). However,  such subclasses still make the RB experiment different from standard execution model.